packages/opc-notification-drawer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@one-platform/opc-notification-drawer",
-  "version": "0.0.1-prerelease",
+  "version": "0.0.2-prerelease",
```

Comment:
@akhilmhdh You are correct, this calls for a bigger change, you can go ahead and make it `0.1.1-prerelease` and could you please also update the `package-lock` as well?

Reply:
Ok